lib/project_types/script/forms/create.rb

```diff
@@ -3,10 +3,11 @@
 module Script
   module Forms
     class Create < ShopifyCli::Form
-      flag_arguments :extension_point, :name
+      flag_arguments :extension_point, :name, :language
 
       def ask
         self.name = valid_name
+        self.language = ask_language
         self.extension_point ||= ask_extension_point
```

Comment:
Just out of curiosity, what's the difference between asking for an extension point and asking for a language? I see that we are memoizing the extension point but not the language.

Reply:
`ask_language` currently validates the argument passed as an option, whereas `ask_extension_point` currently trusts the input and fails later on if the EP doesn't exist. Ideally IMO, `ask_extension_point` should also validate the input but I can add that in later